### ISSUE TITLE
Users and registration

### ISSUE THREAD

**Issue by pehala** [PhilippvK/playforia-minigolf#53]
On Playforia there was the ability to register and then login into the game, of course with this rewrite it is not possible, instead users choose their password after connecting to the server. Do we want to add the ability to register and login?

**Comment by PhilippvK** [comment]
Personally, mainly for privacy reasons (e.g. GDPR), I have no interest in such feature. I am open for suggestions anyway.

**Comment by pehala** [comment]
I totally agree, I think it doesn't make sense in this environment. But the disadvantage is that currently, users can't really reserve their name and it won't allow us to have administrators.

If you are ok with this, I won't consider any users for now (especially in the Database design) and I will leave this thread open for when we (or anybody else) come up with any suggestions.
